awscdk-minecraft/src/zip-directory.py

```diff
@@ -0,0 +1,40 @@
+import os
+from zipfile import ZipFile
+
```

Comment:
I suggest using `from pathlib import Path` and using that instead of strings, since this way it'll be OS-agnostic.

Reply:
that said I suppose this component of the architecture will "always" run on linux so, you can choose to not bother.

Reply:
Ooh, `pathlib.Path` is the best! +1 to this

Reply:
I was unaware of this library. Sounds like a great idea that is supported by the rest of the team.